### PR TITLE
[TW-778] Flows second round edits

### PR DESCRIPTION
**Pull request by petewitcher**

### CODE REVIEW

File: src/pages/docs/getting-started/introduction.md [hunk 1]
```diff
@@ -106,6 +107,13 @@ If you're exposing an API for public use, Postman can support developer onboardi
 * [Publishing documentation](/docs/publishing-your-api/documenting-your-api/)
 * [Using Run in Postman](/docs/publishing-your-api/run-in-postman/introduction-run-button/)
 
+### Creating API workflows
+
+If you're creating an API workflow, you can use Postman Flows:
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/introduction.md:112]: @SterlingChin we're open to whatever description you want to put here on the first page.

**Review by jkonrath-postman — APPROVED** [review]
LGTM! Let's find out if Sterling wants anything else on the first page, but otherwise it looks good to go.

File: src/pages/docs/postman-flows/flows-query-language/data-manipulation.md [hunk 2]
```diff
@@ -7,30 +7,30 @@ You can use the [Flows Query Language](/docs/postman-flows/flows-query-language/
...
+* [Join an array of strings into a single string](#join-an-array-of-strings-into-a-single-string)
+* [Replace string with another](#replace-string-with-another)
+* [Base64 encode a string](#base64-encode-a-string)
+* [Base64 decode a string](#base64-decode-a-string)
+* [Encode a url component](#encode-a-url-component)
+* [Decode a url component](#decode-a-url-component)
+* [Encode an entire url](#encode-an-entire-url)
+* [Decode entire url](#decode-entire-url)
+* [Append to an array](#append-to-an-array)
+* [Time and Date formatting](#time-and-date-formatting)
```
Comment (mark-anthony-postman) [inline src/pages/docs/postman-flows/flows-query-language/data-manipulation.md:33]:
```suggestion
* [Time and date formatting](#time-and-date-formatting)
```

File: src/pages/docs/postman-flows/flows-query-language/data-manipulation.md [hunk 3]
```diff
@@ -7,30 +7,30 @@ You can use the [Flows Query Language](/docs/postman-flows/flows-query-language/
...
+* [Split a string into an array of components](#split-a-string-into-an-array-of-components)
+* [Join an array of strings into a single string](#join-an-array-of-strings-into-a-single-string)
+* [Replace string with another](#replace-string-with-another)
+* [Base64 encode a string](#base64-encode-a-string)
+* [Base64 decode a string](#base64-decode-a-string)
+* [Encode a url component](#encode-a-url-component)
+* [Decode a url component](#decode-a-url-component)
+* [Encode an entire url](#encode-an-entire-url)
+* [Decode entire url](#decode-entire-url)
+* [Append to an array](#append-to-an-array)
```
Comment (mark-anthony-postman) [inline src/pages/docs/postman-flows/flows-query-language/data-manipulation.md:32]: You skipped 4 heading 2's between "Decode entire url" and "Append to an array"

File: src/pages/docs/postman-flows/flows-query-language/data-manipulation.md [hunk 4]
```diff
@@ -7,30 +7,30 @@ You can use the [Flows Query Language](/docs/postman-flows/flows-query-language/
...
+* [Transform a string to all uppercase](#transform-a-string-to-all-uppercase)
+* [Transform a string to all lowercase](#transform-a-string-to-all-lowercase)
+* [Trim a string](#trim-a-string)
+* [Pad a string](#pad-a-string)
+* [Split a string into an array of components](#split-a-string-into-an-array-of-components)
+* [Join an array of strings into a single string](#join-an-array-of-strings-into-a-single-string)
+* [Replace string with another](#replace-string-with-another)
+* [Base64 encode a string](#base64-encode-a-string)
+* [Base64 decode a string](#base64-decode-a-string)
+* [Encode a url component](#encode-a-url-component)
```
Comment (mark-anthony-postman) [inline src/pages/docs/postman-flows/flows-query-language/data-manipulation.md:28]: Use all caps for all cases of URL on this page (except in code blocks)

File: src/pages/docs/postman-flows/remote-execution/webhooks.md [hunk 5]
```diff
@@ -13,15 +13,15 @@ To create a Flow that executes on the Postman servers, create a webhook endpoint
 * [Create a release](#create-a-release)
```
Comment (mark-anthony-postman) [inline src/pages/docs/postman-flows/remote-execution/webhooks.md:13]: On line 12, change "Testing Flows locally" to "Testing flows locally"

**Review by mark-anthony-postman — APPROVED** [review]
Great updates! Thanks for continue to polish the Flows section 💎✨ It's really looking good now! 😀

File: src/pages/docs/postman-flows/core-concepts/blocks-list.md [hunk 6]
```diff
@@ -3,106 +3,124 @@ title: "List of blocks"
...
+* **For**
 
-  Assigns a value to a variable that can then be accessed with the **Get Variable** block anywhere in the Flow.
+  Loops over each item in a list.
 
-- **Get Variable**
+* **Collect**
 
-  Retrieves the value of a previously-assigned variable.
+  Collects the results of a for loop, and outputs a list and an event when it has finished.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/postman-flows/core-concepts/blocks-list.md:110]: Or maybe **for** should be bold, based on how **if** block is bolded on other pages?

File: src/pages/docs/postman-flows/remote-execution/webhooks.md (hunk 5 above)
Comment (petewitcher) [reply src/pages/docs/postman-flows/remote-execution/webhooks.md]: File deleted

**Pull request merged**